murdock/database/mongodb.py

```diff
@@ -28,7 +29,7 @@ async def init(self):
             [("creation_time", pymongo.ASCENDING)], name="job_creation_time"
         )
 
-    def close(self):
+    async def close(self):
```

Comment:
This is marked as async but nothing is awaited in this function. Is it normal ?

Reply:
I changed it to `async` in the abstract class as some database backend (asyncpg) have an `async` close method. Is it fine to remove the `async` with mongodb even though the interface specifies that the function is `async`?

Reply:
I'm not sure but I think this is fine to keep async. Will check later.